scripts/downsample_curriculum.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import logging
+from os.path import isdir
+from pathlib import Path
+from shutil import copytree, rmtree
+from typing import Mapping, Sequence
+from random import sample, seed
+
+import yaml
+
+situation_num: int = 0
+
+
+def get_situation_num(situation_string: Path) -> int:
+    """Given a Path for a situation directory, return the associated number"""
+    return int(str(situation_string).split('_')[-1])
+
+
+def copy_directory(start_path: Path, end_path: Path) -> None:
+    """Function to recursively copy directory from start_path to end_path"""
+    copytree(str(start_path), str(end_path))
+
+
+def copy_all_directories(output_dir: Path, dirs_to_move: Sequence[Path]):
+    """Move all directories in dirs_to_move to their corresponding location in the target directory"""
+    global situation_num
```

Comment:
Is there a reason `situation_num` has to be global here? It doesn't seem to be used anywhere else.